simbad/util/__init__.py

```diff
@@ -55,6 +56,11 @@ def output_files(run_dir, result, output_pdb, output_mtz):
     shutil.copyfile(input_mtz, output_mtz)
 
 
+def python_version():
+    """Return python version"""
+    return sys.version_info.major
```

Comment:
no, no, no, no. please just use wherever necessary
```
if sys.version_info.major < 3:
    # py2 import
else:
    # py3 import
```

Reply:
Yeah I wasn't sure about this and I thought you'd be sure to correct me